01-node-tutorial/answers/11-fs-async.js

```diff
@@ -0,0 +1,26 @@
+const { readFile, writeFile } = require('fs');
+
+readFile('./content/first.txt', 'utf-8', (err, result) => {
+  if (err) {
+    console.log(err);
+    return;
+  }
+  console.log(result);
+});
+
+writeFile('./temporary/fileB.txt', 'This is line 1\r\n', { flag: 'a' }, (err, result) => {
+  if (err) {
+    console.log(err);
```

Comment:
you can add console.log statements at various points in your code to tell you when each step complete